pkg/plugins/golang/v2/api.go

```diff
@@ -165,6 +165,7 @@ func (p *createAPISubcommand) PostScaffold() error {
 		if err != nil {
 			return err
 		}
+		fmt.Printf("TODO: Edit the API definition & reconciliation business logic\nNext: generate targets with:\n$ make manifests\n")
```

Comment:
I am not sure if the text here is clear enough. WDYT about something more similar to the other places (e.g. [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v3/init.go#L154)) such as:

```suggestion
		fmt.Print("Next: implement your new API and generate the manifests (e.g. CRDs,CRs) with:\n$ make manifests \n")
```

It is not passing in the CI because of the size of the line. You can test it locally by running `make lint`. 
Also, ensure that you executed `make testata-generated` before pushing the changes. 
Then, can you also rebase your commits?

Reply:
Thanks for the comments. The changes have been updated as per the sample. Also CI checks done manually now.